types/blob-to-buffer/blob-to-buffer-tests.ts

```diff
@@ -0,0 +1,6 @@
+import * as blobToBuffer from "blob-to-buffer";
```

Comment:
Classes and functions cannot be imported using `import * as` syntax. Please fix either the test or the definition file to use a correct import form.

See also https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this

Reply:
@RyanCavanaugh can you please explain how to fix definition?
Btw its working fine for me as is.